intTests/test1980/test.saw

```diff
@@ -0,0 +1,8 @@
+enable_experimental;
+m <- mir_load_module "test.linked-mir.json";
+
+f8 <- mir_extract m (mir_find_name m "test::f" [mir_u8]);
```

Comment:
What happens when you call `mir_find_name m "foo" []`, where `foo` is a function that takes no type parameters? That might also be worth testing.